cannot find module for type declaration, in v3-next-major-spec

#### Describe the bug
I installed the `v3-next-major-spec` version in Glee and got [these](https://github.com/asyncapi/glee/actions/runs/6429597242/job/17458986740?pr=523) errors. 

```
node_modules/@asyncapi/parser/esm/models/v3/asyncapi.d.ts:12:39 - error TS2307: Cannot find module 'models/schemas' or its corresponding type declarations.

12 import type { SchemasInterface } from 'models/schemas';
                                         ~~~~~~~~~~~~~~~~

```

#### How to Reproduce
I am facing this issue in this [GitHub Action](https://github.com/asyncapi/glee/actions/runs/6429597242/job/17458986740?pr=523).


#### Expected behavior
`models/schemas` should be a local reference.

--- a/src/models/v3/asyncapi.ts
+++ b/src/models/v3/asyncapi.ts
@@ -26,12 +26,12 @@ import type { MessageInterface } from '../message';
 import type { ComponentsInterface } from '../components';
 import type { SecuritySchemesInterface } from '../security-schemes';
 import type { ExtensionsInterface } from '../extensions';
-import type { SchemaInterface } from 'models/schema';
-import type { SchemasInterface } from 'models/schemas';
+import type { SchemaInterface } from '../schema';
+import type { SchemasInterface } from '../schemas';
 import type { v3 } from '../../spec-types';
-import { OperationInterface } from 'models/operation';
-import { ChannelInterface } from 'models/channel';
-import { ServerInterface } from 'models/server';
+import { OperationInterface } from '../operation';
+import { ChannelInterface } from '../channel';
+import { ServerInterface } from '../server';
 
 export class AsyncAPIDocument extends BaseModel<v3.AsyncAPIObject> implements AsyncAPIDocumentInterface {
   version(): string {

--- a/src/models/v3/operation-reply.ts
+++ b/src/models/v3/operation-reply.ts
@@ -2,7 +2,7 @@ import { BaseModel } from '../base';
 import { Channel } from './channel';
 import { Message } from './message';
 import { Messages } from './messages';
-import { MessagesInterface } from 'models/messages';
+import { MessagesInterface } from '../messages';
 import { OperationReplyAddress } from './operation-reply-address';
 
 import { extensions } from './mixins';

--- a/src/models/v3/operation.ts
+++ b/src/models/v3/operation.ts
@@ -11,7 +11,7 @@ import type { ChannelsInterface } from '../channels';
 import type { MessagesInterface } from '../messages';
 import type { MessageInterface } from '../message';
 import type { OperationInterface, OperationAction } from '../operation';
-import type { OperationReplyInterface } from 'models/operation-reply';
+import type { OperationReplyInterface } from '../operation-reply';
 import type { OperationTraitsInterface } from '../operation-traits';
 import type { ServersInterface } from '../servers';
 import type { ServerInterface } from '../server';

